cmake/conan/profiles/linux-clang_libc++

```diff
@@ -0,0 +1,16 @@
+[settings]
+os=Linux
+os_build=Linux
+arch=x86_64
+arch_build=x86_64
+compiler=clang
+# Using Clang 11 so that no issues arise in GitHub CI, but any version newer than that can be used just fine
```

Comment:
Do you mean clang lower than 11 fails in CI?

Reply:
I mean that since the default Clang version in CI is Clang 11, setting it to a higher value would probably cause issues with CI and therefore fail the build due to the Conan preset requiring a higher Clang version than what CI actually uses.

Reply:
Yup that make sense.